src/utils/transformToHarRequest.js

```diff
@@ -0,0 +1,15 @@
+export function transformToHarRequest(data) {
+  const harObject = {
+    method: data.method,
+    url: new URL(data.path, 'https://service.centrapay.com/').href,
+    httpVersion: 'HTTP/2.0',
+    headers: data.headers && Object.entries(data.headers).map(([name, value]) => ({ name, value })),
```

Comment:
@MeganSteenkamp Do we still have the issue where we can't include variables in our curl command? 

Can we do anything to solve it here?

Reply:
@tom-blake yeah unfortunately we can only generate with single quotes.
I can't find a solution to address this.

Reply:
Outcome: Discussed this and changed the Api Key variable to `<TOKEN>` which makes it clearer that you need to put your variable in.